src/components/close-button/__tests__/CloseButton.test.tsx

```diff
@@ -37,4 +37,12 @@ describe('components/close-button/CloseButton', () => {
             expect(handleClick).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe('accesability properties', () => {
+        test('should have an accesiblity label', () => {
```

Comment:
could we move this into the `render()` describe above? this test probably doesn't need to be in its own `describe`

Reply:
I moved it.